packages/shared/src/components/modals/squads/PostModerationModal.tsx

```diff
@@ -67,6 +67,7 @@ function PostModerationModal({
       {...modalProps}
       size={Modal.Size.Large}
       kind={Modal.Kind.FlexibleTop}
+      className="tablet:max-h-[calc(100vh-5rem)]"
```

Comment:
`Modal.Kind.FlexibleTop` have no max-h, maybe we should add a new kind with this restriction?

Reply:
Can you check what we have for `PostModal` and maybe reuse the same configuration we have there.

Reply:
@sshanzel It seems that `BasePostModal` is using _React Portal_ + _custom CSS_, so I don’t think we should proceed with this approach for the moderation modal.

What’s odd is that we only have max-h for `ModalSize.Medium` in the general classes applied to `<Modal>`. Maybe we would like to handle this with a refactor or a general fix in a dedicated task. What do you think?

```
[ModalKind.FlexibleTop]: {
    [ModalSize.XSmall]: 'laptop:mt-16 laptop:mb-10',
    [ModalSize.Medium]: 'min-h-[25rem] tablet:max-h-[calc(100vh-10rem)]',
    [ModalSize.Large]: 'laptop:mt-16 laptop:mb-10',
    [ModalSize.XLarge]: 'laptop:mt-16 laptop:mb-10',
  },
```

Reply:
> @sshanzel It seems that BasePostModal is using React Portal + custom CSS, so I don’t think we should proceed with this approach for the moderation modal.

Yeah, I mean can you check the values used there on how much was the calculated max-h, because they should roughly be the same, it's just a matter where the PostModal was defined in the CSS file rather than the `classname`.

As for the general classes applied to Modals, that is part of the guideline  when that was introduced, if there is any discrepancy, it should be raised to the design team first as this is also the values they use for their components.

Reply:
`tablet:max-h-[calc(100vh-5rem)]` is the same used in other occurrences ✔️